### PR TITLE
Added cmake of snapshot module

### PR DESCRIPTION
**Pull request by juha-h**

### CODE REVIEW

**Comment by juha-h** [comment]
Don't know why macos build failed.

**Comment by sreimers** [comment]
> [@juha-h ](/juha-h)
> ` [Added cmake of g726 module](/baresip/baresip/pull/2025/commits/d39524798bc016f100191dbd8f02e76ecb732c80)`

Looks like g726 (already merged) is also part of this PR.

File: cmake/modules.cmake [hunk 1]
```diff
@@ -73,12 +74,16 @@ endif()
 if(OPUS_FOUND)
   list(APPEND MODULES opus)
 endif()
+if(PNG_FOUND)
+  add_subdirectory(modules/snapshot)
```
Comment (sreimers) [inline cmake/modules.cmake:78]: This has to be `list(APPEND MODULES snapshot)`

**Comment by juha-h** [comment]
I'll try again.

**Pull request closed without merging**